diff --git a/node_modules/@expo/image-utils/build/Cache.js b/node_modules/@expo/image-utils/build/Cache.js
index 85adb45..a6364ac 100644
--- a/node_modules/@expo/image-utils/build/Cache.js
+++ b/node_modules/@expo/image-utils/build/Cache.js
@@ -12,7 +12,7 @@ exports.clearUnusedCachesAsync = clearUnusedCachesAsync;
 const crypto_1 = __importDefault(require("crypto"));
 const fs_1 = __importDefault(require("fs"));
 const path_1 = require("path");
-const CACHE_LOCATION = '.expo/web/cache/production/images';
+const os = require('os'); const CACHE_LOCATION = require('path').join(os.tmpdir(), 'expo-image-cache');
 const cacheKeys = {};
 // Calculate SHA256 Checksum value of a file based on its contents
 function calculateHash(filePath) {
